examples/pubsub_controller/pubsub_controller.c

```diff
@@ -178,6 +178,22 @@ do_stats_display(struct rte_mbuf *pkt) {
         }
 }
 
+static void
+sent_to_nf(uint64_t event_id, void *pkt)
```

Comment:
rename this function `send_event`.  Currently it is too easily confused with our existing functions which let you send to one specific NF.

Reply:
Instead of taking a `void*` as the argument, use the `onvm_event_data` type. This way you won't have to create another one of these inside the `sending_event_data` function and can use the one that was already passed to the function which calls this.

Reply:
The send_event_data() is defined in onvm/onvm_nflib/onvm_event.h and called in several places(such as flow_tracker, etc), the arguments include (event_id, dest_id, void *pkt). Send_event(uint64_t event_id, void *pkt) is defined in examples/pubsub_controller/pubsub_controller.c. In this function, it needs to find the dest_id it will send. So I write the send_event() to find the dest_id and calls send_event_data() to send the final packet. 
Also, I used the void * rather than onvm_event_data type because I needn't to create the onvm_event_data in everywhere. By doing this, I think the code can be more concise and clearly.